lib/Parse/ParseDecl.cpp

```diff
@@ -1269,18 +1271,20 @@ Parser::parseDifferentiatingAttribute(SourceLoc atLoc, SourceLoc loc) {
         SyntaxKind::DeprecatedDerivativeRegistrationAttributeArguments);
     {
       // Parse the name of the function.
-      SyntaxParsingContext FuncDeclNameContext(
-          SyntaxContext, SyntaxKind::FunctionDeclName);
+      // TODO(TF-1058): Make `@differentiating` attribute support qualified
```

Comment:
```suggestion
      // TODO(TF-1058): Make `@derivative` attribute support qualified
```

Reply:
I believe the comment is acceptable as is because it lives in `Parser::parseDifferentiatingAttribute`. `@differentiating` is deprecated and all code will go away with the next release (TF-999).